demo_nodes_py/setup.py

```diff
@@ -43,7 +43,8 @@
             'async_param_client = demo_nodes_py.parameters.async_param_client:main',
             'set_parameters_callback = demo_nodes_py.parameters.set_parameters_callback:main',
             'introspection = demo_nodes_py.services.introspection:main',
-            'matched_event_detect = demo_nodes_py.events.matched_event_detect:main'
+            'matched_event_detect = demo_nodes_py.events.matched_event_detect:main',
+            'use_logger_service = demo_nodes_py.logging.use_logger_service:main'
```

Comment:
```suggestion
            'use_logger_service = demo_nodes_py.logging.use_logger_service:main',
```

(just so that next time we add a demo, we don't have to change 2 lines)

Reply:
Fixed it.